server/pictureCrud.js

```diff
@@ -0,0 +1,124 @@
+const { Picture } = require('../database/dbConnection.js');
+
+const buildApiResponse = (dbRecord, imgType) => {
+  const apiResponse = {};
+  apiResponse.product_id = dbRecord.product_id;
+  apiResponse.image_urls = [];
+
+  dbRecord[imgType].forEach((image) => {
+    apiResponse.image_urls.push(image.img_url);
+  });
+  return apiResponse;
+};
+
+// Retrieves all pictures from the database
+exports.getPictures = async function getPicturesFromDatabase(req, res) {
+  const { productId } = req.params;
+  console.log(productId);
+  try {
+    // const result = await Picture.findOne({ product_id: productId }).exec();
+    const result = await Picture.findOne({ product_id: productId}).exec();
+    console.log(result);
+    res.status(200)
+    res.send(buildApiResponse(result, 'images'));
```

Comment:
Is this to retrieve all image urls - pictures and reviews module? I believe it is just sending the pictures modules urls, not the reviews module URLs.

Reply:
Correct -- this path is just to solely send the picture modules urls. Are you suggesting to adjust the query to only grab that column?